Applications/Games/Origin/Local (Legacy)/script.js

```diff
@@ -19,18 +19,19 @@ new LocalInstallerScript()
         );
     })
     .postInstall(function (wine, wizard) {
-        var originDir = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "/Origin/";
+        const originDir = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "/Origin/";
 
-        new Downloader()
+        const originUpdate= new Resource()
             .wizard(wizard)
             .url("https://origin-a.akamaihd.net/Origin-Client-Download/origin/live/OriginUpdate_9_12_0_34172.zip")
             .checksum("c4a2a742f966efa0114bf8025699007ebbda4d8f")
-            .to(originDir + "OriginUpdate_9_12_0_34172.zip")
+            .name("OriginUpdate_9_12_0_34172.zip")
             .get();
 
         new Extractor()
             .wizard(wizard)
-            .archive(originDir + "OriginUpdate_9_12_0_34172.zip")
+            .archive(originUpdate)
             .to(originDir)
             .extract();
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4764979)